test/integration/scenarios/stress/0.transfer.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(params) {
 	describe('postTransactions @slow', () => {
 		var transactions = [];
 		var maximum = 1000;
+		var waitForExtraBlocks = 10; // Wait for extra blocks to ensure all the transactions are included in the block
```

Comment:
@ManuGowda @MaciejBaj Shouldn't we investigate why this requires now and previously working fine without extra blocks?